pilot/copytool/rucio.py

```diff
@@ -44,7 +44,7 @@ def copy_in(files):
         # stdout, stderr = process.communicate()
         # exit_code = process.poll()
 
-        exit_code, stdout, stderr = execute(executable)
+        exit_code, stdout, stderr = execute(" ".join(executable))
```

Comment:
Might not be necessary after I manage to check-in my latest version. The execute() can deal with both string and lists.